aws-sdk-go-v2/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+ARG GO_VERSION
+FROM public.ecr.aws/docker/library/golang:${GO_VERSION}
```

Comment:
Are there any golang processes actually running in this container or is it only java? I am wondering if we should just use the Amazon Corretto official images directly https://gallery.ecr.aws/amazoncorretto/amazoncorretto